inc/theme-hooks.php

```diff
@@ -110,3 +110,63 @@ function alcatraz_output_footer_bottom() {
 		);
 	}
 }
+
+add_action( 'alcatraz_footer', 'alcatraz_output_social_network_icons', 80 );
+/**
+ * Output the social network icons.
+ *
+ * @since 1.0.0
+ */
+function alcatraz_output_social_network_icons() {
+
+	$options = get_option( 'alcatraz_options' );
+	?>
+
+	<div class="alcatraz-social-icon-wrap">
+		<ul class="alcatraz-social-icons">
+			<?php if ( isset( $options['email_url'] ) ) : ?>
+				<li class="email">
+					<a href="<?php echo esc_url( $options['email_url'] ) ; ?>" class="alcatraz-social-icon alcatraz-icon-email" target="_blank">
+						<i class="fa fa-envelope"></i>
```

Comment:
I think we need a system for both customizing the icon classes and customizing the social networks we include. I realize that we've hardcoded these all over the place so far, so it would be a lot of work. I'm thinking of an alcatraz_get_social_networks function with a more complex array than Mm Components that gets filtered. It would look something like:

```
$social_networks = array(
    'facebook' => array(
        'url' => '',
        'icon_classes' => 'fa fa-facebook-f',
        'description' => __( 'Enter your Facebook profile URL', 'alcatraz' ),
    ),
    'twitter' => array(
        'url' => '',
        'icon_classes' => 'fa fa-twitter',
    ),
);
```

etc. It would be great if by simply tacking new networks on using the filter we suddenly have new fields and icons we can use all over the place. I'm happy to assist with the effort to get such a system in place.